packages/core/src/components/alert/alert.tsx

```diff
@@ -69,7 +69,7 @@ export class Alert extends AbstractPureComponent<IAlertProps, {}> {
         return (
             <Dialog className={classNames(Classes.ALERT, className)} isOpen={isOpen} style={style}>
                 <div className={Classes.ALERT_BODY}>
-                    <Icon iconName={iconName} iconSize="inherit" intent={Intent.DANGER} />
+                    <Icon iconName={iconName} width={40} intent={Intent.DANGER} />
```

Comment:
You never want to set width and height on an icon, only the size of its bounding box, so what about a `size` prop instead?

Reply:
i think `size` is likely the way to go cuz we want them to be squared. 
borrowed this pattern from researching other SVG icons, but it doesn't work super well.